### PR TITLE
dropdown and dropdown ui

### PR DESCRIPTION
**Pull request by miToe**
In this PR the ticket
[Dropdown UI](https://github.com/users/miToe/projects/2/views/1?pane=issue&itemId=65754620) is solved and the bugfix ticket [Bug fix: Received true for a non-boolean attribute end](https://github.com/users/miToe/projects/2/views/1?pane=issue&itemId=66773202) is also solved.

### CODE REVIEW

File: src/components/styles/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+export const DropdownLabel = styled.label`
+  display: block;
+  margin-bottom: 8px;
+  font-size: 14px;
+  color: #333;
+`;
+
+export const IconWrapper = styled.span`
+  display: inline-block;
+  display: flex;
```
Comment (SabinePfanner) [inline src/components/styles/Dropdown.js:18]: I'm not 100% sure, but I guess you can only have one option for ```display:```and not define it twice

File: src/components/Dropdown.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    (event) => {
+      const currentIndex = options.findIndex(
+        (option) => option === event.target.innerText
+      );
+      if (event.key === "ArrowDown" || event.key === "ArrowUp") {
+        event.preventDefault();
+        const offset = event.key === "ArrowDown" ? 1 : -1;
+        const nextIndex =
+          (currentIndex + offset + options.length) % options.length;
+        document.getElementById(`option-${nextIndex}`)?.focus();
```
Comment (SabinePfanner) [inline src/components/Dropdown.js:81]: You're using the old method to access elements via document quite often. In React this should be avoided.

**Review by SabinePfanner — COMMENTED** [review]
In general it looks fine BUT you're using ```document.querySelector```and ```document.getElementByID" quite often - this should be avoided in React.

File: src/components/Dropdown.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  const [focusedIndex, setFocusedIndex] = useState(-1);
+
+  useEffect(() => {
+    function handleClickOutside(event) {
+      if (isOpen && !event.target.closest(".dropdown-container")) {
+        setIsOpen(false);
+      }
+    }
+
+    document.addEventListener("mousedown", handleClickOutside);
```
Comment (mariiaovs) [inline src/components/Dropdown.js:34]: the "mousedown" event is only relevant for desktop devices, for mobile devices you need to handle touch events.

**Review by mariiaovs — COMMENTED** [review]
LGTM! I just left you a small comment.

**Pull request merged**